js/views/modals/purchase/Purchase.js

```diff
@@ -174,7 +184,14 @@ export default class extends BaseModal {
           buildRefreshAlertMessage(app.polyglot.t('purchase.purchaseDataChangedPopin')),
       });
 
-      this.listenTo(this.dataChangePopIn, 'clickRefresh', () => (this.render()));
+      this.listenTo(this.dataChangePopIn, 'clickRefresh', () => {
```

Comment:
IMHO, it doesn't really work well to pass in a modsChanged flag because multiple data changes may have caused the pop up to show. For example, the localCurrency may have changed and then a minute later, the verified mods might have changed.

I think we could simplify by making refresh re-render both things, e.g:

```
      this.listenTo(this.dataChangePopIn, 'clickRefresh', () => {
          this.render();
          this.moderators.render();
      });
```